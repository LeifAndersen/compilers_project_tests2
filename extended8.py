def a(bob:"bleh"=3): pass

