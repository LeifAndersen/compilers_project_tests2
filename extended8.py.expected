(program (def (a ("=" bob 3)) (pass)))

